### PR TITLE
Add Epidemic QA Topic Files and TopicReader

### PR DESCRIPTION
**Pull request by justinborromeo**
Follow up to #1393 that adds a TopicReader for Epidemic QA.

### CODE REVIEW

File: src/test/java/io/anserini/search/topicreader/EpidemicQATopicReaderTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.SortedMap;
+
+import static org.junit.Assert.assertEquals;
+
+public class EpidemicQATopicReaderTest
+{
```
Comment (lintool) [inline src/test/java/io/anserini/search/topicreader/EpidemicQATopicReaderTest.java:13]: braces with above line, please fix everywhere.

File: src/test/java/io/anserini/search/topicreader/EpidemicQATopicReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  @Test
+  public void test() throws IOException
+  {
+    TopicReader<Integer> consumer_reader =
+        new EpidemicQATopicReader(
+            Paths.get("src/main/resources/topics-and-qrels/topics.epidemic-qa.consumer.prelim.json"));
+
+    SortedMap<Integer, Map<String, String>> consumer_topics = consumer_reader.read();
```
Comment (lintool) [inline src/test/java/io/anserini/search/topicreader/EpidemicQATopicReaderTest.java:21]: stick to java camelCase convetions? e.g., `consumerTopics`?

File: src/test/java/io/anserini/search/topicreader/TopicReaderTest.java [hunk 3]
```diff
@@ -726,4 +727,57 @@ public void testBackgroundLinkingTopics() {
...
+
+    // No consumer questions from CQ035 to CQ037
+    assertEquals(42, consumer_topics.keySet().size());
+    assertEquals(1, (int) consumer_topics.firstKey());
+    assertEquals("what is the origin of COVID-19",
+                 consumer_topics.get(consumer_topics.firstKey()).get("question"));
+    assertEquals("CQ001", consumer_topics.get(consumer_topics.firstKey()).get("question_id"));
+    assertEquals("coronavirus origin", consumer_topics.get(consumer_topics.firstKey()).get("query"));
+    assertEquals("seeking information about whether the virus was designed in a lab or occured "
+                 + "naturally in animals and how it got to humans",
```
Comment (lintool) [inline src/test/java/io/anserini/search/topicreader/TopicReaderTest.java:744]: I think our convention is `+` on previous line?

Yea, we should install a linter, I know...

**Comment by justinborromeo** [comment]
Addressed comments.

**Review by lintool — APPROVED** [review]

**Pull request merged**